### DISCUSSION TITLE
Eu Escolho Você!

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#314] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/tirinhas/eu-escolho-voce/

**Comment by brunoaugusto** [comment]
Por que eu tô rindo tanto disso? :rofl:

**Comment by one-eyed-willy** [comment]
Eu fiquei imaginando o Professor Carvalho andando de lado que nem o Krabby :rofl:
